engines/mbic3.cpp

```diff
@@ -676,7 +676,7 @@ size_t ModelBasedIC3::push_blocking_clause(size_t i, Term c)
 
   Result r;
   size_t j;
-  for (j = i; j < frames_.size(); ++j) {
+  for (j = i; j + 1 < frames_.size(); ++j) {
```

Comment:
can you check this @makaimann ?

Reply:
Yep that looks good to me! Should be equivalent to the previous fix but this is much cleaner, thanks